src/api_server/swagger/firecracker.yaml

```diff
@@ -1179,7 +1179,8 @@ definitions:
         type: string
         description:
           The microVM version for which we want to create the snapshot.
-          It is optional and it defaults to the current version.
+          It is optional and it defaults to the current version. This parameter
+          has been deprecated and it will be removed in future Firecracker release.
```

Comment:
maybe specify the firecracker version when it was deprecated?

Reply:
I thought about it but (1) other similar messages don't mention it and (2) after thinking about it, I don't think it's really useful information. But I don't have strong opinions

Reply:
Ok, I don't have a strong preference either.